tests/openbsd_wifi.pp

```diff
@@ -0,0 +1,4 @@
+bsd::network::interface::wifi { 'wpi0':
+network_name => 'myssid',
+address      => [ 'dhcp', ],
```

Comment:
Two things I think are causing test failure.  Indent line 2 and 3 and remove the trailing comma inside the array.